app/src/test/java/com/example/marvelapp/MainViewModelTest.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.marvelapp
```

Comment:
The ViewModel tests should be inside a ViewModel package

Reply:
fixed